can_bus/quadruna/INVR/INVR_tx.json

```diff
@@ -939,24 +939,6 @@
             }
         }
     },
-    "DebugReadWriteParamCommand": {
```

Comment:
removing this because CAN ID 83 is reserved for inverter canRx, inverter CAN protocol is that it receives a command with CAN ID 83 and response with a msg with CAN ID 84